test/test_cpp_extensions_jit.py

```diff
@@ -529,6 +531,38 @@ def compile(code):
         module = compile("int f() { return 789; }")
         self.assertEqual(module.f(), 789)
 
+    @unittest.skipIf(codecs.lookup(locale.getencoding()).name != "utf-8", "Only test in UTF-8 locale")
```

Comment:
(As [getencoding](https://docs.python.org/3/library/locale.html#locale.getencoding) as added in 3.11+)
```suggestion
    @unittest.skipIf('utf' not in locale.getlocale()[1].lower(), "Only test in UTF-8 locale")
```